.github/workflows/cypress.yaml

```
@@ -0,0 +1,148 @@
+      - develop
+
+jobs:
+  Install:
+    runs-on: ubuntu-latest
+    env:
+        REACT_APP_TALAWA_URL: http://localhost:4000/graphql
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v4 
+
+      - name: Cypress Install 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)